Satellite-im/UplinkWeb
# chore(profile): add support button

### What this PR does 📖

- Adds support button on profile

![image](https://github.com/user-attachments/assets/e1fc4985-4ea4-4007-b0c5-51e2fbaee7cc)


how to test
- profile should be the same
- hovering should show the email
- clicking on it should open whatever email app you use
